src/System.Management.Automation/engine/InternalCommands.cs

```diff
@@ -415,7 +415,7 @@ private void InitParallelParameterSet()
                             this));
                 }
             }
-
+            
```

Comment:
@PaulHigin It seems you don't configure your editor to automatically truncate trailing spaces.